gradle/libs.versions.toml

```diff
@@ -45,6 +45,9 @@ kotlinpoet-metadata-core = { module = "com.squareup:kotlinpoet-metadata", versio
 kotlinpoet-metadata-specs = { module = "com.squareup:kotlinpoet-metadata-specs", version.ref = "kotlinpoet" }
 kotlinxMetadata = "org.jetbrains.kotlinx:kotlinx-metadata-jvm:0.3.0"
 okio = "com.squareup.okio:okio:2.10.0"
+guava = "com.google.guava:guava:30.1.1-jre"
+moshi = { module = "com.squareup.moshi:moshi", version.ref = "moshi" }
+moshiAdapters = { module = "com.squareup.moshi:moshi-adapters", version.ref = "moshi" }
```

Comment:
Let's call these something more specific to indicate they are just for japicmp. Same with the moshi version itself

Reply:
What do you mean ? Can you just give me suggest changes ?